### PR TITLE
PHP Doc and such

### PR DESCRIPTION
**Pull request by imkingdavid**

### CODE REVIEW

File: phpBB-Blog/includes/blog.php [hunk 1]
```diff
@@ -54,19 +66,35 @@ public function __construct($id = 0)
...
 	public function setId($id = 0)
 	{
 		$this->id = (int) $id ?: $this->id;
 		return $this;
 	}
 
+	/**
+	* Retrieve array of data for a blog post
+	*
+	* @return current object instance
```
Comment (erikfrerejean) [inline phpBB-Blog/includes/blog.php]: Please add the `type`, so proper docs can be generated

Comment (erikfrerejean) [inline phpBB-Blog/includes/blog.php]: http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.return.pkg.html

Comment (imkingdavid) [inline phpBB-Blog/includes/blog.php]: type is the long class name. _sigh_ guess I'll be copy/pasting a lot :P

Comment (erikfrerejean) [inline phpBB-Blog/includes/blog.php]: Don't blame me for thinking off way to long classnames ;)

Comment (imkingdavid) [inline phpBB-Blog/includes/blog.php]: lol actually it's required for the class loader, so we don't have to do includes and such.

Comment (erikfrerejean) [inline phpBB-Blog/includes/blog.php]: I know that ;), but still I'm unblamable for that :p

**Pull request merged**